util_windows.go

```diff
@@ -93,10 +93,30 @@ func readFileOnce(filename string) ([]byte, error) {
 		// cheaply. If the size was wrong, we'll either waste some space off the end
 		// or reallocate as needed, but in the overwhelmingly common case we'll get
 		// it just right.
-		if size := fi.Size() + bytes.MinRead; size > n {
-			n = size
+		n += fi.Size()
+	}
+
+	data := make([]byte, n)
+	remaining := data
+	var total int64
+	for {
```

Comment:
You can `io.Copy` into a `bytes.Buffer` instead. That'll handle growing and everything else. `bytes.Buffer` implements `ReadFrom` with code that's pretty much identical to the code you have here: https://cs.opensource.google/go/go/+/refs/tags/go1.19:src/bytes/buffer.go;l=198-215

Reply:
Yeah mb, I've miss red the docs, I've seen `unred` and thought this returned the extra capacity, but it's infact red from the buffer's point of view, not the file's one.